documentation.md

```diff
@@ -1,11 +1,13 @@
 ---
 layout: page
-title: Documentation
-permalink: /docs/
+title: Docs
+permalink: /documentation/
+external-link: //pelux.io/software-factory
```

Comment:
Perhaps you should remove the permalink and the content of this page if this never will be visible because there is no link to it anymore?